opentelemetry-otlp/src/retry.rs

```diff
@@ -144,13 +128,13 @@ where
 
                 match error_type {
                     RetryErrorType::NonRetryable => {
-                        otel_warn!(name: "OtlpRetry", message = format!("Operation {:?} failed with non-retryable error: {:?}", operation_name, err));
+                        otel_warn!(name: "OtlpRetryNonRetryable", operation = operation_name, error = format!("{:?}", err));
                         return Err(err);
                     }
                     RetryErrorType::Retryable if attempt < policy.max_retries => {
                         attempt += 1;
                         // Use exponential backoff with jitter
-                        otel_warn!(name: "OtlpRetry", message = format!("Retrying operation {:?} due to retryable error: {:?}", operation_name, err));
+                        otel_info!(name: "OtlpRetryRetrying", operation = operation_name, error = format!("{:?}", err));
```

Comment:
the `error` here is going to be the stringified/formatted one, likely coming from the underlying transport library, partially defeating structured logging purpose.

Reply:
also, this is a retry attempt  and need to log the delay/jitter etc. to be fully useful.

Reply:
> the error here is going to be the stringified/formatted one, likely coming from the underlying transport library, partially defeating structured logging purpose.

Do you have a suggestion here?

Reply:
> > the error here is going to be the stringified/formatted one, likely coming from the underlying transport library, partially defeating structured logging purpose.
> 
> Do you have a suggestion here?

Yes. Don't log the inner error in the retry module.